src/main/java/com/hierynomus/msdfsc/ReferralCache.java

```diff
@@ -219,6 +224,21 @@ ReferralCacheEntry getReferralEntry(Iterator<String> pathComponents) {
             return ENTRY_UPDATER.get(this);
         }
 
+        void deleteExpiredReferralEntry(Iterator<String> pathComponents) {
+            if (this.entry != null && this.entry.isExpired() &&
+                !this.entry.isRoot() /*&& !this.entry.isLink()*/) {
```

Comment:
remove the commented out code. add a comment if necessary.

Reply:
Done.